src/test/perl/test-caffileeditor_source.t

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib "$Bin/modules";
```

Comment:
apologies again, you need code like https://github.com/quattor/CAF/blob/master/src/test/perl/kerberos-process.t#L4
the dummy LC modules are in the resources dir